ash/src/extensions_generated.rs

```diff
@@ -16198,8 +16242,6 @@ pub mod khr {
             pub cmd_pipeline_barrier2_khr: PFN_vkCmdPipelineBarrier2,
             pub cmd_write_timestamp2_khr: PFN_vkCmdWriteTimestamp2,
             pub queue_submit2_khr: PFN_vkQueueSubmit2,
-            pub cmd_write_buffer_marker2_amd: PFN_vkCmdWriteBufferMarker2AMD,
-            pub get_queue_checkpoint_data2_nv: PFN_vkGetQueueCheckpointData2NV,
```

Comment:
Technically a semver-breaking change too. These functions were provided by `VK_AMD_buffer_marker` or `VK_NV_device_diagnostic_checkpoints` respectively, but `vk.xml` previously exposed them under the `VK_KHR_synchronization2` extension, and so did our generator.